assets/test/view_test.js

```diff
@@ -104,6 +107,8 @@ describe('View', function() {
     view.displayError();
     expect(el.classList.contains('phx-disconnected')).toBeTruthy();
     expect(el.classList.contains('phx-error')).toBeTruthy();
+    expect(el.classList.contains('phx-connected')).toBeFalsy();
+    expect(el.classList.contains('user-implemented-class')).toBeTruthy();
     expect(loader.style.display).toEqual('block');
```

Comment:
Tests updated to show that:

1) `phx-connected` is never around when `phx-disconnected` is.
2) `user-implemented-class` remains untouched throughout common operations.